### DISCUSSION TITLE
Limit which libraries users can see

### DISCUSSION THREAD

**Discussion by SulliL** [sct/overseerr#2149] [Feature Requests]
So here is a hypothetical. I have 3 plex libraries I have imported into overseerr. Plex User A only has access too 2 of those libraries through Plex but as of right now in overseerr they can still see the stuff in the library they don't have access to as being available.

It would prob be difficult to have Plex automatically grab this information and limit what users can see as available in overseerr so since overseerr already knows what is in each library adding the ability to toggle which libraries each user could see on a per user basis would be really nice.

**Comment by TheCatLady** [comment]
This isn't possible, as Overseerr does not keep track of where items are located or the libraries that users have access to.
